packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -47,6 +24,68 @@ WrapWith.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
+const isReactLink = (alienElement) =>
```

Comment:
same case as previously described. Need to check if a React element is a `Link`. I do it by checking if it contains an `href` prop